sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -362,22 +317,38 @@ async function resolveConflictsToTheirs(git: SimpleGit, commitDetails: ICommitDe
         let conflicted = status.conflicted[lp];
         let fileStatus = getFileStatus(status, conflicted);
         let fileState = `${fileStatus.index.padEnd(1)}${fileStatus.working_dir.padEnd(1)}`.replace(/\s/g, "_");
+
         commitSummary[fileState] = (commitSummary[fileState] + 1 || 1);
+
         if (fileStatus.index === "D") {
             // index      workingDir     Meaning
             // -------------------------------------------------
             // D                  deleted from index
             // D             D    unmerged, both deleted
             // D             U    unmerged, deleted by us
-            commitMessage = logAppendMessage(commitMessage, fileStatus, "Removed from theirs");
-            await git.rm(conflicted);
+            if (fileStatus.working_dir === "U") {
```

Comment:
This was one identified case where the previous logic for DU (deleted by us) during a merge was incorrectly deleting the file. The deleted by us was because a commonly named file was matched by git merge but we actually "moved" the file to `auto-merge/xxxx` folder.